lib/jsdom/browser/Window.js

```diff
@@ -198,8 +199,10 @@ function Window(options) {
     }
     return startTimer(window, setInterval, clearInterval, ++latestTimerId, fn, ms, args);
   };
+  this.requestAnimationFrame = fn => this.setTimeout(fn, 1000 / 60, performanceNow());
```

Comment:
Hmm I don't think performanceNow() is correct. Per https://w3c.github.io/hr-time/#time-origin-0 it should be when the jsdom window is created, whereas this package uses "the reported time is relative to the time the current Node process has started". These are not the same.

I think instead you need to record the appropriate (high-res) time in the Window constructor.

Reply:
Good point!